koku/api/report/ocp/query_handler.py

```diff
@@ -278,9 +278,8 @@ def execute_query(self):  # noqa: C901
                 query_data = self._project_classification_annotation(query_data)
             if self._limit and query_data:
                 query_data = self._group_by_ranks(query, query_data)
-                if not self.parameters.get("order_by") or self.parameters.get("order_by", {}).get(
-                    "cost_total_distributed"
-                ):
+                order_by = self.parameters.get("order_by")
+                if not order_by or set(order_by.keys()).intersection(["cost_total", "cost_total_distributed"]):
```

Comment:
A dict iterates over the keys by default. There is no need to get the keys view specifically in order to build a `set`
```suggestion
                if not order_by or set(order_by).intersection(["cost_total", "cost_total_distributed"]):
```